app/controllers/documents_controller.rb

```diff
@@ -3,11 +3,19 @@
 # Document management
 class DocumentsController < ApplicationController
   def index
+    @sorting_params = [params[:sort], params[:direction]]
+    @category = params[:category]
+    join_sql = "INNER JOIN active_storage_attachments ON active_storage_attachments.record_id = documents.id INNER JOIN active_storage_blobs ON active_storage_blobs.id = active_storage_attachments.blob_id"
+
+    respond_to do |format|
+      format.js
+      format.html
+    end
```

Comment:
Since you don't pass any options to `.js` and `.html` options, I think you can omit this block entirely. The controller will by default render the action-matching `.js` template when it receives a JS request